src/components/Navbar/Navbar.js

```diff
@@ -17,18 +30,29 @@ const Navigation = ({ toggleHandler }) => {
           alt="logo"
         ></img>
       </Link>
-      <div className="nav-spacer ">
-        <SearchBar />
-      </div>
+      <button
+        className="set-lang"
+        onClick={(event) => {
+          onClickLang();
+          switchImage();
```

Comment:
this is not too talkative. if someone read this part it is hard to understand what is happening here.
"switchImage" what image? it could be 'switchLanguageFlagImage' or something like this
same with "onClickLang" it could be switchLanguage

so if someone come to this part can understand it like: "okay if someone click on this button we will switch the language and switch the image flag on the navber"
I hope you get my point